source/style-guide/style/version.txt

```diff
@@ -0,0 +1,140 @@
+.. _version-numbers:
+
+==================
+Version Numbers
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. |add| replace:: `versionadded <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded>`__
+.. |change| replace:: `versionchanged <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged>`__
+.. |deprecated| replace:: `deprecated <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated>`__
+
+Versions are unique names assigned to product releases that are packaged 
+with a set of features at a specific point in time. 
+MongoDB products follow semantic versioning (link to wikipedia). 
```

Comment:
Issue:
Looks like the link is missing.

Suggestion:
Use the following software versioning wikipedia link:
https://en.wikipedia.org/w/index.php?title=Software_versioning&oldid=1147438049